### PR TITLE
Docs for using secret in TDengine TelemetryService.

### PR DESCRIPTION
**Pull request by FFFFFaraway**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Add Docs about how to use the TDengine TelemetryService with password in secret.
corresponding PR: https://github.com/Edgenesis/shifu/pull/421

#### Which issue(s) this PR fixes:

Fixes #

#### How is this PR tested:
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/guides/telemetryservice/pushtoTDEngine.md [hunk 1]
```diff
@@ -28,10 +28,21 @@ spec:
...
 - `dbType` is the type of the database
   
+
 :::note
 If you have multiple telemetry services, you can write them in one file and split them in one file using `---`.
 :::
 
+:::note use secret to store password
+
+Create a  `Secret` with `telemetry_service_sql_pwd` field filled by your password, and it will overwrite the `SQLSetting.secret` in `TelemetryService`.
```
Comment (kris21he) [inline docs/guides/telemetryservice/pushtoTDEngine.md:38]: can we also add the process(from UI to backend) of how to store the secret and how to get that secret in this k8s Secret service?

Comment (FFFFFaraway) [reply docs/guides/telemetryservice/pushtoTDEngine.md]: ok, now have added the process of how to get and use `Secret`.

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/telemetryservice/pushtoTDEngine.md [hunk 2]
```diff
@@ -31,6 +31,31 @@ spec:
...
+kubectl get secret -n deviceshifu
+```
+
+在`deviceshifu-deployment.yaml`中关联该`Secret`：
+
+```yaml
+- name: deviceshifu-secret
+  secret:
+		secretName: deviceshifu-secret
+		optional: true
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/telemetryservice/pushtoTDEngine.md:54]: same as above

File: docs/guides/telemetryservice/pushtoTDEngine.md [hunk 3]
```diff
@@ -28,10 +28,36 @@ spec:
...
+
+```bash
+kubectl get secret -n deviceshifu
+```
+
+Associate the `Secret` in `deviceshifu-deployment.yaml`:
+
+```yaml
+- name: deviceshifu-secret
+  secret:
```
Comment (tomqin93) [inline docs/guides/telemetryservice/pushtoTDEngine.md:54]: Also need to specify the secret mount here

**Review by tomqin93 — COMMENTED** [review]
@FFFFFaraway added some comment, please take a look

File: docs/guides/telemetryservice/pushtoTDEngine.md [hunk 4]
```diff
@@ -15,19 +15,27 @@ spec:
     SQLSetting:
       serverAddress: 192.168.14.163:6041
       username: root
-      secret: taosdata
+      secret: my-secret
```
Comment (tomqin93) [inline docs/guides/telemetryservice/pushtoTDEngine.md:18]: need to update to include username injection

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@FFFFFaraway added a comment, please take a look

File: docs/guides/telemetryservice/pushtoTDEngine.md (hunk 4 above)
Comment (FFFFFaraway) [reply docs/guides/telemetryservice/pushtoTDEngine.md:18]: ok

Comment (tomqin93) [inline docs/guides/telemetryservice/pushtoTDEngine.md:18]: @FFFFFaraway do we still need username if we specify username in secret? if not we need to explain it here

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**